### ISSUE TITLE
Suggestion: Shift Transport Rings block check up one layer

### ISSUE THREAD

**Issue by ShicKla** [MrJake222/AUNIS#193]
As of now the transport rings block ignores the first layer above it. If you can have it ignore the first 2 layers above it, it would allow players to place decorations at the surface and still be able to bury the block.

**Comment by MrJake222** [comment]
I believe it already checks 2 blocks... :thinking: For ex. snow layers don't get destroyed. Or at least they didn't few versions ago

**Comment by ShicKla** [comment]
It will not replace 2 blocks height above it, but when it checks to see if it will be obstructed it will consider some blocks as obstructions. Such as torches, picture frames, and Little Tiles.

**Comment by MrJake222** [comment]
So the replacing is good but checking is skewed?

**Comment by ShicKla** [comment]
That is correct. Snow layer does not seem to be consider an obstruction, but a lot of other blocks are. If I disable the obstruction check then decorations on the surface are not replaced, but anything higher up is, as intended.
![Untitled Diagram](https://user-images.githubusercontent.com/2679084/82322765-47617400-9994-11ea-9630-1bcb67d5806c.png)

**Comment by MrJake222** [comment]
So maybe a check similar to beamers? Because rings flying through stone would be weird. But transparent blocks could do

**Comment by ShicKla** [comment]
That would be reasonable. Most decorations that I used, and have seen others use would be considered transparent blocks.
